README.md

```diff
@@ -62,6 +62,21 @@ To run individual end-to-end tests, use `yarn` inside the `api_tests` folder:
 - `yarn run fix`: run prettier and linter to fix format
 - `yarn run check`: run tsc (to check validity of TypeScript code) and verify format
 
+## Tor
+
+If you would like to maintain anonymity while using cnd for atomic swaps we support running cnd over Tor.
+You will need to configure an onion service (previously hidden service), for example:
```

Comment:
The "for example" might be misleading in that it suggests that you can freely choose the port whereas in reality, cnd is hardcoded against 9939 for the hidden service.

Reply:
Good point, I was only referring to the virtual port. Will make it clearer, thanks.